pkg/actor/actor.go

```diff
@@ -41,17 +42,26 @@ type Actor interface {
 	// We choose message to have a concrete type instead of an interface to save
 	// memory allocation.
 	Poll(ctx context.Context, msgs []message.Message) (running bool)
+
+	// Close is called after Poll returns false,
+	// or actor system is stopping and all message has been received.
+	//
+	// Close must be idempotent and nonblocking.
+	Close()
```

Comment:
1.  for normal cases, resources can be released before `Poll` returns `false`.
2. The name `Close` confused me,  it sounds like we can Close the actor directly, but it's just a callback.